docker/entrypoint.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 set -ex
-
+export NODE_ENV=production
```

Comment:
Why do you need to set it in `Dockerfile` and here in `entrypoint.sh`?

Reply:
looks like it's redundant, I've removed it in https://github.com/ghostfolio/ghostfolio/pull/3614/commits/482c99144e3916e707417ab4957d130f057f7a91